package.json

```diff
@@ -68,8 +68,8 @@
     "pretty-quick": "^2.0.1",
     "solc": "0.5.8",
     "ts-jest": "^26.5.1",
-    "ts-node": "^8.3.0",
-    "typescript": "^3.8.3"
+    "ts-node": "10.7.0",
+    "typescript": "4.1.6"
```

Comment:
no carret here to force this version. 4.2 is not yet compatible with valora's codebase